cache/in-memory/src/model/message.rs

```diff
@@ -129,7 +129,12 @@ impl CachedMessage {
         self.application_id
     }
 
-    /// Attached files.
+    /// List of attached files.
+    ///
+    /// Refer to the documentation for [`Message::content`] for caveats with
```

Comment:
should this be `Message::content` or `Message::attachments`? cf. https://github.com/twilight-rs/twilight/pull/1677/files#diff-9301b8d7b30e8a99b05e29f920273ebcc28a727c9983c1ea1d53347dd545605eR20